go.mod

```diff
@@ -4,21 +4,25 @@ go 1.20
 
 require (
 	github.com/go-git/go-git/v5 v5.6.1
+	github.com/google/go-github v17.0.0+incompatible
+	github.com/google/go-github/v54 v54.0.0
```

Comment:
You shouldn't need this dependency: run `go mod tidy`

Reply:
You'll also want to rebase on the main branch to get the latest changes in the go module ☁️